runtime/protocol/http-client-engines/http-client-engine-okhttp4/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+description = "OkHttp 4.x Client Engine for Smithy services generated by smithy-kotlin"
+extra["displayName"] = "Smithy :: Kotlin :: HTTP :: Engine :: OkHttp4"
+extra["moduleName"] = "aws.smithy.kotlin.runtime.http.engine.okhttp4"
+
+kotlin {
+    sourceSets {
+        commonMain {
+            dependencies {
+                api(project(":runtime:protocol:http-client"))
+                implementation(project(":runtime:protocol:http-client-engines:http-client-engine-okhttp"))
+                implementation(libs.okhttp4)
+            }
+        }
+        commonTest {
+            dependencies {
+                implementation(libs.kotlinx.coroutines.test)
+            }
+        }
+
+        jvmTest {
+            dependencies {
+                implementation(project(":runtime:testing"))
+                // use otel testing capabilities
+                implementation(project(":runtime:observability:telemetry-provider-otel"))
+                implementation(libs.opentelemetry.sdk.testing)
+            }
+        }
```

Comment:
**Question**: Why do we have these dependencies for `commonTest` and `jvmTest`? I can't see a test source set of any kind in this module.

Reply:
They are left over from my copy-paste of the OkHttp `build.gradle.kts`, will remove